lib/exchange_api_web/controllers/order_controller.ex

```diff
@@ -2,9 +2,61 @@ defmodule ExchangeApiWeb.OrderController do
   use ExchangeApiWeb, :controller
 
   def index(conn, _params) do
-    json(conn, %{status: "** shrug das orders **"})
+    orders = Exchange.open_orders(get_ticker())
+    json(conn, %{ status: elem(orders, 0), data: elem(orders, 1) })
   end
 
-  def init(_foo) do
+  def create(conn, %{"side" => side, "size" => size, "price" => price,  "type" => type, "exp_time" => time}) do
+    exp = elem(DateTime.from_iso8601(time), 1)
```

Comment:
same here, you can do: `exp = DateTime.from_iso8601!(time)` or `{:ok, exp} = DateTime.from_iso8601(time)`

better than both of these would be to deal with the case where this is an error and responding with it

Reply:
Still trying to figure out how to make the conversions from. milliseconds. Will handle this as soon as possible.